sources/models/src/aws-ecs-1/mod.rs

```diff
@@ -5,7 +5,7 @@ use std::collections::HashMap;
 use crate::modeled_types::Identifier;
```

Comment:
I see the new setting in ecs, aws 1.19, and vmware, but shouldn't it also be in aws 1.21, and *-dev?  (Meaning all variants, when you account for symlinks.)

Reply:
All those `mod.rs` are symlinked to one of the `mod.rs` for aws-ecs-1, aws-k8s-1.19 or vmware-k8s-1.21. I didn't add it for the *-dev variants since I thought they didn't need it but on second thought they probably should have it.